function_calling/config.yaml

```diff
@@ -50,4 +50,35 @@ tools:
         "k_retrieved_documents": 3
         "score_treshold": 0.3
 
-prod_mode: False
\ No newline at end of file
+prod_mode: False
+
+st_tools: # set which tools to show in the streamlit app set up bar
+  "get_time":
+    "enabled": True
+    "default": True
+  "query_db":
+    "enabled": True
+    "default": True
+  "python_repl":
+    "enabled": True
+    "default": True
+  "calculator":
+    "enabled": True
+    "default": False
+  "translate":
+    "enabled": True
+    "default": False
+  "rag":
+    "enabled": True
+    "default": False
+
+st_preset_queries: 
+  "Create a summary table in the db": >
+    Create and save a table in the database that will show the top 10 albums with the highest
+    sales in 2013 and in the USA. The table fields will be the name of the album, the name of the
+    artist, the total amount of sales, and the number of copies sold.
+  "Get information of the created summary table": >
+    Give me a summary of the 2013 top albums table
+  "Create insightful plots of the summary table": >
+    Get the information of the 2013 top albums table in the DB, when you get the data then create
+    some meaningful plots that summarize the information, and store them in PNG format
```

Comment:
Do we really need this in config.yaml? I would keep it in app.py. I dont think users will change this part.

Reply:
@snova-petrojm Yes we do. This is for easier application to hosted instances via yaml and not .py files, and follows our standardization of this application across other kits. We should add an explanation though.

Reply:
@snova-kwasia i dont think this is a good place for this and will confuse developers. The config.yaml file should contain input params and settings that configure how the code operates. Those sample questions only impact the UI design. I would place it in a different yaml file

Reply:
@snova-kwasia i dont think this is the best place for this and may confuse developers. The config.yaml input file should contain input params and settings that configure how the code and model operate. Those sample questions only impact the UI design. I would suggest placing them in a different yaml file, maybe similar to what we did for EKR and other kits, under prompts/*.yaml. Just my thoughts. Happy to discuss more.

Reply:
Ok understood @snova-petrojm and makes sense. Let's use prompts.yaml @snova-jorgep.

Reply:
preset queries moved to `/prompts/streamlit_preset_queries.yaml`